lib/core/src/lib/datatable/components/datatable/datatable.component.scss

```diff
@@ -229,6 +229,8 @@ $data-table-cell-min-width-file-size: $data-table-cell-min-width !default;
         box-sizing: border-box;
 
         .adf-datatable-row {
+            padding-right: 30px;
```

Comment:
Did you test this change by building control center with ADF containing this change? I've just tried it and it looks like this padding breaks the layout
<img width="1722" alt="Zrzut ekranu 2023-07-7 o 11 38 58" src="https://github.com/Alfresco/alfresco-ng2-components/assets/113341662/0b91e722-21d8-4527-84e8-f13eab4a3978">

Reply:
Yes, I did do that. There has been some change, which got merged recently which is causing this. I am rebasing and checking it as we speak.

Reply:
The code has been updated to accommodate the new merge which was https://github.com/Alfresco/alfresco-ng2-components/pull/8671. It is working fine now.